api.planx.uk/modules/flows/copyFlowAsPortal/service.ts

```diff
@@ -7,7 +7,6 @@ import { Flow } from "../../../types";
 const copyPortalAsFlow = async (flowId: string, portalNodeId: string) => {
   // fetch the parent flow data
   const flow = await getFlowData(flowId);
-  if (!flow) throw Error("Unknown flowId");
```

Comment:
This was redundant and unreachable, this exception is already thrown inside `getFlowData()`